lib/backend/k8s/conversion/conversion_test.go

```diff
@@ -1049,6 +1054,10 @@ var _ = Describe("Test NetworkPolicy conversion (k8s <= 1.7, no policyTypes)", f
 			pol, err = c.K8sNetworkPolicyToCalico(&np)
 			Expect(err).NotTo(HaveOccurred())
 			Expect(pol.Key.(model.ResourceKey).Name).To(Equal("knp.default.testPolicy"))
+			Expect(pol.Key.(model.ResourceKey).Namespace).To(Equal("default"))
+			Expect(pol.Value.(*apiv2.NetworkPolicy).Name).To(Equal("knp.default.testPolicy"))
+			Expect(pol.Value.(*apiv2.NetworkPolicy).Namespace).To(Equal("default"))
+			Expect(int(*pol.Value.(*apiv2.NetworkPolicy).Spec.Order)).To(Equal(1000))
```

Comment:
Looks like a duplicate Expect here

Reply:
one is checking the key, the other the metadata